src/SystemCommands-MessageCommands/SycHierarchicalSendersCommand.class.st

```diff
@@ -0,0 +1,45 @@
+"
+This command find the senders of a given method in the classes upper in the inheritance chain above a given method.
+"
+Class {
+	#name : 'SycHierarchicalSendersCommand',
+	#superclass : 'SycChangeMessageSignatureCommand',
```

Comment:
It does look like senders is a kind of change signature command

Reply:
The class documentation of SycChangeMessageSignatureCommand says: "I am a base class for commands which modify signature of given message". However I don't see how querying the hierarchical senders would change anything. Could you elaborate?